packages/flutter_tools/lib/runner.dart

```diff
@@ -128,18 +132,52 @@ Future<int> _handleToolError(
       command: args.join(' '),
     );
 
-    if (error is String) {
-      stderr.writeln('Oops; flutter has exited unexpectedly: "$error".');
-    } else {
-      stderr.writeln('Oops; flutter has exited unexpectedly.');
-    }
+    final String command = _crashCommand(args);
+
+    final String errorString = error.toString();
+    printError(errorString);
 
     try {
-      final File file = await _createLocalCrashReport(args, error, stackTrace);
-      stderr.writeln(
-        'Crash report written to ${file.path};\n'
-            'please let us know at https://github.com/flutter/flutter/issues.',
-      );
+      final String doctorText = await _doctorText();
```

Comment:
Hopefully the doctor won't crash, but I think it would be good to guard against it separately and continue with printing the github issue instructions.

Reply:
doctor exceptions are already caught in `_doctorText`.